projection/raw_transaction/raw_transaction_integration_test.go

```diff
@@ -0,0 +1,135 @@
+package raw_transaction_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/crypto-com/chain-indexing/appinterface/rdb"
+	rdb_test "github.com/crypto-com/chain-indexing/appinterface/rdb/test"
+	event_entity "github.com/crypto-com/chain-indexing/entity/event"
+	. "github.com/crypto-com/chain-indexing/entity/event/test"
+	entity_projection "github.com/crypto-com/chain-indexing/entity/projection"
+	. "github.com/crypto-com/chain-indexing/external/logger/test"
+	"github.com/crypto-com/chain-indexing/external/primptr"
+	"github.com/crypto-com/chain-indexing/external/utctime"
+	"github.com/crypto-com/chain-indexing/infrastructure/pg"
+	"github.com/crypto-com/chain-indexing/projection/block"
+	block_view "github.com/crypto-com/chain-indexing/projection/block/view"
+	. "github.com/crypto-com/chain-indexing/test"
+	event_usecase "github.com/crypto-com/chain-indexing/usecase/event"
+	model_usecase "github.com/crypto-com/chain-indexing/usecase/model"
+)
+
+var _ = Describe("RawTransaction", func() {
+	It("should implement projection", func() {
+		fakeLogger := NewFakeLogger()
+		fakeRdbConn := rdb_test.NewFakeRDbConn()
+		var _ entity_projection.Projection = block.NewBlock(fakeLogger, fakeRdbConn, nil)
+	})
+
```

Comment:
This is some old naive tests I have back in the days. A better way could be just have a line in the source code file:
```
var _ entity_projection.Projection = &RawTransaction{}
```